rootfs-builder/rootfs.sh

```diff
@@ -79,6 +82,29 @@ check_function_exist() {
 	[ "$(type -t ${function_name})" == "function" ] || die "${function_name} function was not defined"
 }
 
+generate_dockerfile() {
+	dir="$1"
+
+	readonly install_go="
+ADD https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.tar.gz /tmp
```

Comment:
Would be nice to make the golang URL configurable

Reply:
Agreed - similar to https://github.com/kata-containers/agent/pull/112.

Reply:
This could be handled by a follow-on PR though as it would be great to get this PR landed today. WDYT @jcvenegas ?

Reply:
@liangchenye @jodh-intel yes, I prefer to have this PR merged today, and then fix the URL and clearlinux container usage.

Reply:
Captured at #47 if someone wants to pick that up.